content/communityPage/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Community Page
+---
+
+ [Getting Started](/download) Get started building an app by reading the quick-start guide.
+
+ Stay up to date by joining our [slack channel](https://slack.openjsf.org).
```

Comment:
Note that OpenJSF Slack is not an official chat space for the project (there are no official chat spaces since we were never able to agree on one), so if we're mentioning the OpenJSF Slack here we should mention IRC and Node Slackers as well.